frontend/components/dashboardTile.js

```diff
@@ -61,9 +61,31 @@ var DashboardTile = createReactClass({
         }
     },
 
+    getDashboardTileStyles: function() {
+        var flexBasis = '31%';
+        var flexGrow = this.props.shouldTilesStretchAcrossRows ? 1 : 0;
+
+        switch (this.props.tilesPerRow) {
+            case 1: {
+                flexBasis = '100%'
+                break;
+            }
+            case 2: {
+                flexBasis = '48%'
+                break;
+            }
+            case 3: {
+                flexBasis = '31%'
+                break;
+            }
+        }
+
+        return {flexBasis: flexBasis, flexGrow: flexGrow}
```

Comment:
Just for future reference - this could be `return {flexBasis, flexGrow}`. If the key and value are the same - it works!!!

Reply:
Oh yes of course! Will make the change :)